packages/console/src/components/Entities/EntityDetails.tsx

```diff
@@ -15,6 +16,11 @@ import { EntityVersions } from './EntityVersions';
 import { EntityExecutionsBarChart } from './EntityExecutionsBarChart';
 
 const useStyles = makeStyles((theme: Theme) => ({
+  entityDetailsWrapper: {
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
```

Comment:
```suggestion
    paddingInline: theme.spacing(2),
```

Reply:
I personally think paddingInline does the same stuff as paddingLeft/right but additionally also takes care of localization. For a RTL language, you might want padding on the right side rather than the left or something similar and since we support internationalization, we should make sure to use apt spacing attributes/values.

Confirmed that padding inline and block work with MUI as well so we should try to use it when possible unless we are explicitly sure that we want to use padding left which didnt seem to be the case in here.